### PR TITLE
Tanium threat response

### PR DESCRIPTION
**Pull request by dantavori**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/17722

## Description
New integration: `Tanium Threat Response`
New playbooks: `Tanium Threat Response - Request File Download` and `Tanium Threat Response - Create Connection`

## Does it break backward compatibility?
  - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
- [ ] Tanium Threat Response - Create Connection
- [ ] Tanium Threat Response - Request File Download

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/e11dd4031bc7b3ff40dce19b47169ae4031c97ce/Integrations/TaniumThreatResponse/TaniumThreatResponse.yml)
- [ ] [README file](https://github.com/demisto/content/blob/e11dd4031bc7b3ff40dce19b47169ae4031c97ce/Integrations/TaniumThreatResponse/README.md)

### CODE REVIEW

**Comment by guyfreund** [comment]
Finished dealing all the issues, code can be reviewed

**Comment by guyfreund** [comment]
> lets demo it, and assign Alex to review this please

I'll assign Alex to review it when documentation will be ready, ATM we have 3 commands which Adi and I don't know how to run, waiting for a response from Dan.

**Comment by guyfreund** [comment]
@kirbles19 yaml & docs are ready to be reviewed

**Comment by ronykoz** [comment]
@kirbles19 can you please check this one out?

**Comment by kirbles19** [comment]
@dantavori @adi88d Done.

**Comment by ronykoz** [comment]
@Arsenikr What fixes do we need to do here?

**Comment by Arsenikr** [comment]
We have a meeting today to sum them up, but from what i saw so far, the changes are not backward compatible so we prefer to wait till the next release

File: Integrations/TaniumThreatResponse/TaniumThreatResponse.py [hunk 1]
```diff
@@ -950,7 +1063,12 @@ def create_evidence(client, data_args):
 def delete_evidence(client, data_args):
     evidence_id = data_args.get('evidence-id')
     client.do_request('DELETE', f'/plugin/products/trace/evidence/{evidence_id}', resp_type='content')
-    return f"Evidence {evidence_id} has been deleted successfully.", {}, {}
+    context = {
+        'ID': int(evidence_id),
```
Comment (ronykoz) [inline Integrations/TaniumThreatResponse/TaniumThreatResponse.py]: why int?

File: Integrations/TaniumThreatResponse/TaniumThreatResponse.py [hunk 2]
```diff
@@ -1006,20 +1124,25 @@ def get_file_download_request_status(client, data_args):
...
+                                     headers=headers, headerTransform=pascalToSpace, removeNull=True)
     return human_readable, outputs, raw_response
 
 
 def delete_file_download(client, data_args):
     file_id = data_args.get('file-id')
     client.do_request('DELETE', f'/plugin/products/trace/filedownloads/{file_id}', resp_type='text')
-    return f"Delete request of file with ID {file_id} has been sent successfully.", {}, {}
+    context = {
+        'ID': int(file_id),
```
Comment (ronykoz) [inline Integrations/TaniumThreatResponse/TaniumThreatResponse.py]: why int?

**Comment by ronykoz** [comment]
@dantavori don't forget to create a readme file

**Pull request closed without merging**